modules/jellyfin/default.nix

```diff
@@ -1,42 +1,40 @@
-{ lib, pkgs, config, ... }:
+{ pkgs, lib, config, ... }:
 with lib;
 let cfg = config.paul.jellyfin;
 in
 {
 
   options.paul.jellyfin = {
     enable = mkEnableOption "activate jellyifn";
+    openFirewall = mkEnableOption "open firewall for jellyfin";
     enableNginx = mkEnableOption "activate nginx proxy";
   };
 
-  config = mkIf cfg.enable {
-
-    systemd.services.jellyfin =
-      let
-        compose-file = ./docker-compose.yml;
-      in
-      {
+  config = mkIf cfg.enable
+    {
+      systemd.services.jellyfin = {
         description = "Jellyfin media server docker-compose service";
         wantedBy = [ "multi-user.target" ];
         after = [ "docker.service" "docker.socket" ];
-        serviceConfig = mkMerge [
-          {
-            User = "root";
-            Group = "root";
-            ExecStart = "${pkgs.docker-compose}/bin/docker-compose -f ${compose-file} up";
```

Comment:
Instead of calling `${pkgs.docker-compose}/bin/docker-compose` we can call `${pkgs.docker}/bin/docker compose`